tests/wpt/metadata/css/css-backgrounds/bg-color-with-gradient.html.ini

```diff
@@ -0,0 +1,2 @@
+[bg-color-with-gradient.html]
```

Comment:
Why did these start failing? The ref uses canvas, is something broken with the canvas bindings or something?

Reply:
I'm not actually sure. The reftest analyzer shows all the pixels differ by one green value (iirc), and it's not clear to me what would cause that.

Reply:
Ah, so fuzziness... Does servo support fuzzy annotations on WPT? It'd be better than marking them as failures.

Reply:
Good point!